example/opencl-print-info.py

```diff
@@ -0,0 +1,63 @@
+"""
+Prints relevant information regarding the capabilities of the current OpenCL runtime and devices
+Note that pyopencl has a script that prints all properties in its examples folder
+"""
+
+import pyopencl as cl
+
+print("PyOpenCL version: " + cl.VERSION_TEXT)
+print("OpenCL header version: " + ".".join(map(str, cl.get_cl_header_version())) + "\n")
+
+# Get installed platforms (SDKs)
+print("- Installed platforms (SDKs) and available devices:")
+platforms = cl.get_platforms()
+
+for plat in platforms:
+    indent = ""
+
+    # Get and print platform info
+    print(indent + "{} ({})".format(plat.name, plat.vendor))
+    indent = "\t"
+    print(indent + "Version: " + plat.version)
+    print(indent + "Profile: " + plat.profile)
+    print(indent + "Extensions: " + str(plat.extensions.strip().split(" ")))
```

Comment:
I would suggest to remove the lines like "Extensions",  "Max work item dims" etc, which do not concern the users but only coders (even not that much). I'll add comments to the lines I think are unnecessary